airbyte-scheduler/persistence/src/test/java/io/airbyte/scheduler/persistence/DefaultJobPersistenceTest.java

```diff
@@ -161,6 +163,10 @@ public void setup() throws Exception {
     database = new JobsDatabaseInstance(container.getUsername(), container.getPassword(), container.getJdbcUrl()).getAndInitialize();
     resetDb();
 
+    DatabaseMigrator jobDbMigrator = new JobsDatabaseMigrator(database, "test");
```

Comment:
@tuliren I'm realising that our current tests don't run Flyway migrations. I think we are going to have to add this to all the config tests too. Am I missing anything?

Will do in a follow up PR.

Reply:
I will look into it and create a follow up PR.

Reply:
Actually this is not unexpected. I know that I will miss updating some tests. So the plan is that we will deal with this when we write the first migration. I thought that I would be that person, but you actually did this for the first time. Sorry about the extra work.

Reply:
no worries!